lib/chef-dk/policyfile_lock.rb

```diff
@@ -15,7 +16,7 @@
 # limitations under the License.
 #
 
-require 'digest/sha1'
+require 'digest/sha2'
```

Comment:
I think I asked this question on the last policyfile review, and came to the same conclusion - the policyfile identifiers are going to be changing.  But that is okay because it won't affect production systems until someone uploads a new policyfile_lock.  Thanks @danielsdeleo!